packages/contract-artifacts/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": {
+        "outDir": "lib",
+        "rootDir": ".",
+        "resolveJsonModule": true
```

Comment:
When I tried to do that I've hit a problem that TSC auto-generates types for JSON files from their content and we had some partial artifacts in some modules, but I guess now that you standardized it  - it's not a problem any more. Happy about that.

Reply:
Yes, I still had to include some type assertions in the `contracts` package for the reasons you mentioned. I think it only works for `contract-artifacts` right now because we are using the trimmed down contract artifacts there.